### ISSUE TITLE
'TypeError: Cannot read property 'name' of undefined'

### ISSUE THREAD

**Issue by grepped** [mentum/lambdaws#36]
I'm getting this error running the example code with my accessKey, secretKey, and created role. 

    /usr/local/lib/node_modules/lambdaws/lib/LambdaHelper.js:182
        var handlerName = configs.name || 'default';
                                 ^
    TypeError: Cannot read property 'name' of undefined
    at getCloudedFunctionFromFunction (/usr/local/lib/node_modules/lambdaws/lib/LambdaHelper.js:182:34)
    at exports.create (/usr/local/lib/node_modules/lambdaws/lib/lambdaws.js:79:30)
    at Object.<anonymous> (/Users/lancestacenko/Development/Projects/50_lambda/lambdaws/test.js:14:17)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
    at startup (node.js:124:16)
    at node.js:842:3

**Comment by slvnperron** [comment]
@grepped Thanks for reporting this issue. Fixed in #39 .. should be merged in master shortly

**Comment by slvnperron** [comment]
@grepped In the mean time just provide lambdaws with a name for the function like so
```js
var cloudedFunction = lambdaws.create(yourFunc, { name: 'FUNCTIONNAME' });
```
